execute/aggregate_test.go

```diff
@@ -171,6 +171,224 @@ func TestAggregate_Process(t *testing.T) {
 				},
 			},
 		},
+		{
+			name:   "two tables one empty",
+			config: execute.DefaultAggregateConfig,
+			agg:    sumAgg,
+			data: []*executetest.Table{
+				{
+					KeyCols: []string{"_start", "_stop"},
+					ColMeta: []flux.ColMeta{
+						{Label: "_start", Type: flux.TTime},
+						{Label: "_stop", Type: flux.TTime},
+						{Label: "_time", Type: flux.TTime},
+						{Label: "_value", Type: flux.TFloat},
+					},
+					KeyValues: []interface{}{
+						execute.Time(0),
+						execute.Time(100),
+					},
+					Data: [][]interface{}{
+						{execute.Time(0), execute.Time(100), execute.Time(0), 0.0},
+						{execute.Time(0), execute.Time(100), execute.Time(10), 1.0},
+						{execute.Time(0), execute.Time(100), execute.Time(20), 2.0},
+						{execute.Time(0), execute.Time(100), execute.Time(30), 3.0},
+						{execute.Time(0), execute.Time(100), execute.Time(40), 4.0},
+						{execute.Time(0), execute.Time(100), execute.Time(50), 5.0},
+						{execute.Time(0), execute.Time(100), execute.Time(60), 6.0},
+						{execute.Time(0), execute.Time(100), execute.Time(70), 7.0},
+						{execute.Time(0), execute.Time(100), execute.Time(80), 8.0},
+						{execute.Time(0), execute.Time(100), execute.Time(90), 9.0},
+					},
+				},
+				{
+					KeyCols: []string{"_start", "_stop"},
+					ColMeta: []flux.ColMeta{
+						{Label: "_start", Type: flux.TTime},
+						{Label: "_stop", Type: flux.TTime},
+						{Label: "_time", Type: flux.TTime},
+						{Label: "_value", Type: flux.TFloat},
+					},
+					KeyValues: []interface{}{
+						execute.Time(100),
+						execute.Time(200),
+					},
+					Data: [][]interface{}{},
+				},
+			},
+			want: []*executetest.Table{
+				{
+					KeyCols: []string{"_start", "_stop"},
+					ColMeta: []flux.ColMeta{
+						{Label: "_start", Type: flux.TTime},
+						{Label: "_stop", Type: flux.TTime},
+						{Label: "_value", Type: flux.TFloat},
+					},
+					Data: [][]interface{}{
+						{execute.Time(0), execute.Time(100), 45.0},
+					},
+				},
+				{
+					KeyCols: []string{"_start", "_stop"},
+					ColMeta: []flux.ColMeta{
+						{Label: "_start", Type: flux.TTime},
+						{Label: "_stop", Type: flux.TTime},
+						{Label: "_value", Type: flux.TFloat},
+					},
+					Data: [][]interface{}{
+						{execute.Time(100), execute.Time(200), nil},
+					},
+				},
+			},
+		},
+		{
```

Comment:
Same thing from above here too. It might be easier to just remove the first table to reduce confusion. I'm not sure it adds anything to the test.